### ISSUE TITLE
[BUG] issue with force-resolutions

### ISSUE THREAD

**Issue by kellyselden** [sonatype-nexus-community/auditjs#240]
**Describe the bug**
Updating from v4.0.25 to v2.0.27 causes:

```
error /xxx/node_modules/auditjs: Command failed.
Exit code: 1
Command: npx force-resolutions
Arguments: 
Directory: /xxx/node_modules/auditjs
Output:
npm ERR! code SELF_SIGNED_CERT_IN_CHAIN
npm ERR! errno SELF_SIGNED_CERT_IN_CHAIN
```

cc @bhamail / @DarthHater / @allenhsieh / @Slim-Shary

**Comment by kellyselden** [comment]
You may want to move force-resolutions from `preinstall` to `prepare`. According to https://docs.npmjs.com/cli/v6/using-npm/scripts, that would run it when `npm install`ing in your repo, but not when consumers `npm install`, which is what I think you want, since you're only affecting your own package-lock.json, and not consumers (since they might have a yarn.lock).

```
npm install
		preinstall
		install
		postinstall
	Also triggers

		prepublish (when on local)
		prepare (when on local)
```

**Comment by DarthHater** [comment]
This should be addressed now on `4.0.29` completely. We switched from `npm` to `yarn` completely, as `yarn` supports the resolutions field natively.

**Comment by kellyselden** [comment]
nice work!
